### PR TITLE
[KED-3052] Add globalToolbarWidth to constants

### PR DESCRIPTION
**Pull request by tynandebold**
## Description

Fixes [KED-3052](https://jira.quantumblack.com/browse/KED-3052).

## Development notes

I've added on the `globalToolbarWidth` const to the necessary widths and removed it from a place where it wasn't needed.

I also capitalize the use of `Plotly` in the release-notes file and formatted it.

## QA notes

Look at our default data set and expand the `Reporting` pipeline. Before this changed one part of the flowchart looked like this:

image

And now we have:

image

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: src/components/flowchart/draw.js [hunk 1]
```diff
@@ -60,10 +59,11 @@ export const drawLayerNames = function () {
     layers,
   } = this.props;
 
+  console.log('sidebarWidth: ', sidebarWidth);
```
Comment (studioswong) [inline src/components/flowchart/draw.js:62]: please delete

**Review by studioswong — COMMENTED** [review]
LGTM on initial review - will check out the branch to test this out.

**Review by studioswong — APPROVED** [review]
Tested this with the same scenario as listed on the ticket, looks much better now.

There is still a slight cut off on the left with the current zoom setting, but imo I don't think it's anything to sweat - putting it here in case we want to fix this later. 
image

File: src/components/flowchart/draw.js (hunk 1 above)
Comment (tynandebold) [reply src/components/flowchart/draw.js:62]: Whoops. Thanks. Deleted.

**Pull request merged**